suite-common/wallet-core/src/send/sendFormEthereumThunks.ts

```diff
@@ -46,31 +46,43 @@ const calculate = (
     feeLevel: FeeLevel,
     token?: TokenInfo,
 ): PrecomposedTransaction => {
-    const feeInSatoshi = calculateEthFee(
-        toWei(feeLevel.feePerUnit, 'gwei'),
-        feeLevel.feeLimit || '0',
-    );
-
     let amount: string;
     let max: string | undefined;
+    const feeInGwei = calculateEthFee(toWei(feeLevel.feePerUnit, 'gwei'), feeLevel.feeLimit || '0');
+
     const availableTokenBalance = token
         ? amountToSmallestUnit(token.balance!, token.decimals)
         : undefined;
     if (output.type === 'send-max' || output.type === 'send-max-noaddress') {
-        max = availableTokenBalance || calculateMax(availableBalance, feeInSatoshi);
+        max = availableTokenBalance || calculateMax(availableBalance, feeInGwei);
         amount = max;
     } else {
         amount = output.amount;
     }
 
     // total ETH spent (amount + fee), in ERC20 only fee
-    const totalSpent = new BigNumber(calculateTotal(token ? '0' : amount, feeInSatoshi));
+    const totalSpent = new BigNumber(calculateTotal(token ? '0' : amount, feeInGwei));
 
     if (totalSpent.isGreaterThan(availableBalance)) {
-        const error = token ? 'AMOUNT_NOT_ENOUGH_CURRENCY_FEE' : 'AMOUNT_IS_NOT_ENOUGH';
+        if (token) {
+            return {
+                type: 'error',
+                error: 'AMOUNT_NOT_ENOUGH_CURRENCY_FEE_WITH_ETH_AMOUNT',
+                errorMessage: {
+                    id: 'AMOUNT_NOT_ENOUGH_CURRENCY_FEE_WITH_ETH_AMOUNT',
+                    values: {
+                        symbol: 'ETH',
+                        feeAmount: `(${fromWei(feeInGwei, 'ether')} ETH)`,
```

Comment:
This method is for all EVMs, why `ETH` is hardcoded?
<img width="726" alt="Screenshot 2024-12-20 at 20 15 02" src="https://github.com/user-attachments/assets/401c4390-4ac0-4df6-bed2-ae72b0aec74b" />